projects/plugins/protect/src/js/hooks/use-protect-data/index.js

```diff
@@ -166,6 +165,5 @@ export default function useProtectData(
 		lastChecked,
 		hasUncheckedItems,
 		jetpackScan,
-		hasRequiredPlan,
```

Comment:
Moved `hasRequiredPlan` out of the protect data hook, and into the more specific `usePlan` hook.